egui/src/widgets/plot/transform.rs

```diff
@@ -217,6 +224,16 @@ impl ScreenTransform {
         Value::new(x, y)
     }
 
+    pub fn rect_from_values(&self, value1: &Value, value2: &Value) -> Rect {
+        let pos1 = self.position_from_value(value1);
+        let pos2 = self.position_from_value(value2);
+        // Note that the transform mirrors rects: top becomes bottom
```

Comment:
I don't understand what this comment is trying to communicate about the surrounding code

Reply:
My guess of the original author's idea would be: the coordinate system for egui has +Y down, but for plots +Y is up. So a rectangle based on `Value`s will be mirrored up-down.

Clarified with Rustdoc:
```rs
/// Transform a rectangle of plot values to a screen-coordinate rectangle.
///
/// This typically means that the rect is mirrored vertically (top becomes bottom and vice versa),
/// since the plot's coordinate system has +Y up, while egui has +Y down. 
pub fn rect_from_values(&self, value1: &Value, value2: &Value) -> Rect { ... }
```